src/hooks/use-edit.ts

```diff
@@ -56,7 +56,6 @@ export const useEdit = ({
 
     if (key.delete || key.backspace) {
       bufferCommands.delete();
-      setIsMSN(true);
```

Comment:
The problem with removing this part is that if I write the high nibble (e.g. press B on currently selected byte), then press delete, then edit the byte under the cursor, it's going to write to the low nibble. I guess adding the buffer to the dependency array of the useEffect should also solve this.

Reply:
Yep, that's right, an additional benefit from that is that we could remove explicit flag setting in switch handling `CommandChar` cases